app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    @post = Post.find(params[:post_id])
+    new_comment = current_user.comments.new(
+      text: comment_params,
+      user_id: current_user.id,
+      post_id: @post.id
+    )
```

Comment:
- [ ] Since you are using the `current_user` to create a comment, you don't need to pass the `user_id` param. Because of this, I got the below error when trying to create a comment. Kindly fix it. Also, test that you are able to create a comment from your end before submitting it for review.
![Screenshot from 2022-02-25 20-43-08](https://user-images.githubusercontent.com/20245164/155762844-5342f5c5-5c56-49f4-8926-51e9e6224cbc.png)